src/components/icon-button/icon-button.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { IconButton } from './icon-button';
+
+// More on how to set up stories at: https://storybook.js.org/docs/react/writing-stories/introduction#default-export
+const meta = {
+  title: 'Example/IconButton',
+  component: IconButton,
+  parameters: {
+    // Optional parameter to center the component in the Canvas. More info: https://storybook.js.org/docs/react/configure/story-layout
+    layout: 'centered',
+  },
+  // This component will have an automatically generated Autodocs entry: https://storybook.js.org/docs/react/writing-docs/autodocs
+  tags: ['autodocs'],
+  // More on argTypes: https://storybook.js.org/docs/react/api/argtypes
+  argTypes: {
+  },
+} satisfies Meta<typeof IconButton>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+// More on writing stories with args: https://storybook.js.org/docs/react/writing-stories/args
+export const Send: Story = {
+  args: {
+    variant: 'send',
+    classNames: 'hover:bg-primary-100',
```

Comment:
:hover zustand sollte in der Komponente selber sein

Reply:
probleme ist wenn du in specs schaust, fast jede icon button hat andere hover effect

Reply:
es gibt nur 1 icon button.  
- normalzustand
- hover zustand
- focus zustand
https://www.figma.com/file/nsXR2h0KwciWpuwKRD58FX/Mumble?type=design&node-id=413-288&mode=design&t=kA1UCI7Jse8yL2io-0